ext_localconf.php

```diff
@@ -1,6 +1,8 @@
 <?php
 
-defined('TYPO3_MODE') or die();
+if (!defined('TYPO3_MODE')) {
```

Comment:
please use "TYPO3" instead of "TYPO3_MODE"

Reply:
👍🏻